pkg/innerring/processors/container/processor.go

```diff
@@ -43,9 +44,20 @@ type (
 		FeeProvider       *config.FeeConfig
 		ContainerClient   *wrapper.Wrapper
 		NeoFSIDClient     *neofsid.ClientWrapper
+		NetworkState      NetworkState
 	}
 )
 
+// NetworkState is an interface of a component
+// that provides access to network state.
+type NetworkState interface {
```

Comment:
Define this interface together with other types, please. We have `AlphabetState` right above, for example.

Reply:
I dont think it is useful since these types are independent.